catboost/libs/train_lib/train_model.cpp

```diff
@@ -50,11 +50,27 @@
 #include <util/random/shuffle.h>
 #include <util/system/compiler.h>
 #include <util/system/hp_timer.h>
-
+#include <library/cpp/threading/local_executor/tbb_local_executor.h>
```

Comment:
remove this line and add #include "create_executor.h" to CatBoost section of include's

Reply:
no longer relevant